WLC/code_executor/abstract_executor.py

```diff
@@ -0,0 +1,89 @@
+import logging
+
+import docker
+import os
+from hackerrank.HackerRankAPI import HackerRankAPI
+from image_segmentation.preprocessor import Preprocessor
+
+from ..utils.azure import WLCAzure
+from ..ocr.picture_ocr import PictureOCR
+
+LOGGER = logging.getLogger()
+
+DEFAULT_DOCKER_PORT = "2375"
+
+
+class AbstractCodeExecutor:
+    NO_OUTPUT = "Program did not generate any output!"
```

Comment:
I have been meaning to mention this but do you think that we should show both the output and the error log? If yes, then this string looks fine. Otherwise we should keep this empty as it will be nicer to check for that in the front-end.

Reply:
I  dont really have a strong opinion on this but I just copied this code over I dont want to make these changes in this MR.